solarforecastarbiter/validation/tasks.py

```diff
@@ -47,7 +52,8 @@ def validate_ghi(observation, values):
         `validator.check_timestamp_spacing`,
         `validator.check_irradiance_day_night`,
         `validator.check_ghi_limits_QCRad`,
-        `validator.check_ghi_clearsky`
+        `validator.check_ghi_clearsky`,
+        `validator.detect_clearsky_ghi`
```

Comment:
not a big deal, but I believe in numpydoc format the tuple is implied by multiple return values, so proper format would be

```
Returns
-------
timestamp_flag
    bitmask from `validator.check_timestamp_spacing`
night_flag
    bitmask from `validator.check_irradiance_day_night`

etc
```